update.sh

```diff
@@ -115,6 +119,9 @@ sed -i 's/SLJIT_CALL3,/SLJIT_CALL, SLJIT_ARG1(SW) | SLJIT_ARG2(SW) | SLJIT_ARG3(
 sed -i 's/sljit_create_compiler(NULL)/sljit_create_compiler(NULL, NULL)/g' src/net/bpfjit.c
 sed -i 's/sljit_emit_enter(compiler, 0, 2,/sljit_emit_enter(compiler, 0, SLJIT_ARG1(SW) | SLJIT_ARG2(SW),/g' src/net/bpfjit.c
 sed -i 's/sljit_free_code((void \*)code)/sljit_free_code((void *)code, NULL)/g' src/net/bpfjit.c
+# Guard for redefines (libpcap compability)
+sed -i 's|\(#define _NET_BPF_H_\)$|\1\n\n/* Avoid redefines when using libpcap */\n#ifndef lib_pcap_bpf_h|g' src/net/bpf.h
+sed -i 's|\(#define[\t ]\+BPF_MEMWORDS[\t ]\+16\)$|\1\n\n#endif /* !lib_pcap_bpf_h */|g' src/net/bpf.h
```

Comment:
These start to look risky. If netbsd reorders some of these, sed may silently hide the wrong things.

Can we save the patch in a file (git diff > patch.diff) and let the update script apply it using git apply instead? Or just have the patch as a commit in this repo? (I don't remember right now how the filter-repo merge stuff works, so I'm not sure what's possible.)

Reply:
Added patchfiles